Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/CardHeader.cpp

```diff
@@ -143,6 +143,13 @@ namespace AzQtComponents
         m_titleLabel->setVisible(!title.isEmpty());
     }
 
+    void CardHeader::setTitle(const QString& title, const QString& tooltip)
```

Comment:
should this method be more specific like setTitleAndTooltip() ?

Reply:
I almost named it that, but I figured the parameter name and there being two different overrides for setTitle would be good enough.

Reply:
I guess would it make sense to just split it into two methods?

Reply:
Good idea, I've split it into two functions to make it more clear